common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -61,17 +61,6 @@ function(_) {
         function Player(el, config) {
             var errorMessage, lastSource, sourceList;
 
-            // A simple test to see that the 'config' is a normal object.
-            if ($.isPlainObject(config) === false) {
-                return;
-            }
```

Comment:
Do you think the control will never come to this point, if config is not plain object ?